certified-connectors/HitHorizons/apiProperties.json

```diff
@@ -0,0 +1,35 @@
+{
+  "properties": {
+    "connectionParameters": {
+      "api_key": {
+        "type": "securestring",
+        "uiDefinition": {
+          "displayName": "API Key",
+          "description": "The API Key for this api",
+          "tooltip": "Provide your API Key",
+          "constraints": {
+            "tabIndex": 2,
+            "clearText": false,
+            "required": "true"
+          }
+        }
+      }
+    },
+    "iconBrandColor": "#8200ff",
+    "capabilities": [],
+    "policyTemplateInstances": [
+      {
+        "templateId": "setheader",
+        "title": "Add PowerApps source information header",
+        "parameters": {
+          "x-ms-apimTemplateParameter.name": "X-Source-PowerApps",
+          "x-ms-apimTemplateParameter.value": "powerapps",
+          "x-ms-apimTemplateParameter.existsAction": "override",
+          "x-ms-apimTemplate-policySection": "Request"
+        }
+      }
+    ],
+    "publisher": "FinStat, s. r. o.",
```

Comment:
@krivulcik Please let us know why `publisher` and `stackOwner` values different from `HitHorizons` ?

Reply:
@sriyen-msft The project HitHorizons.com doesn't have its own legal entity. FinStat, s. r. o. is the company running it. Originally, the company ran only https://www.finstat.sk/ , recently, we started the project https://www.hithorizons.com/ . More information available here: https://www.hithorizons.com/about-us , also company information available in T&C: https://www.hithorizons.com/legal/terms-and-conditions and Privacy Policy: https://www.hithorizons.com/legal/privacy-policy .